doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml

```diff
@@ -12,8 +12,12 @@ data:
         Path_Key true
```

Comment:
Can you open a PR to add a RayCluster YAML to the KubeRay repo and reference the YAML in this doc?

Example:
https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/configuring-autoscaling.html#step-3-create-a-raycluster-custom-resource-with-autoscaling-enabled